src/deepsparse/server/server.py

```diff
@@ -64,6 +70,10 @@ def start_server(
     server_config = ServerConfig(**obj)
     _LOGGER.info(f"Using config: {repr(server_config)}")
 
+    if hot_reload_config:
+        _LOGGER.info(f"Watching {config_path} for changes.")
+        _ = start_config_watcher(config_path, f"http://{host}:{port}/endpoints", 0.5)
```

Comment:
we can probably afford to check less frequently than 0.5s no?

Reply:
It's pretty cheap to check even every half second. Prefer a more responsive watcher